Error thrown when current directory has no package.json

When using claycli in a directory with no `package.json` you get the following error:

```
$ clay --version
module.js:538    throw err;
    ^

Error: Cannot find module '/CURRENT_DIR/package.json'    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)    at require (internal/module.js:11:18)
    at Object.<anonymous> (/redact/.nvm/versions/node/v8.9.0/lib/node_modules/claycli/node_modules/amphora-fs/lib/getComponents/index.js:11:9)
    at Module._compile (module.js:635:30)    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
```

This is likely cause by `amphora-fs` which [requires the file outside of a function call](https://github.com/clay/amphora-fs/blob/5509a196b497198410a33a23c4084935b6affec2/lib/getComponents/index.js#L11) so the fix might have to happen there.  Alternatively if no fix is possible then it should be signaled in the README that you must run claycli from a folder with a `package.json`

--- a/lib/cmd/compile/media.js
+++ b/lib/cmd/compile/media.js
@@ -8,11 +8,7 @@ const _ = require('lodash'),
   changed = require('gulp-changed'),
   es = require('event-stream'),
   destPath = path.join(process.cwd(), 'public', 'media'),
-  mediaGlobs = '*.+(jpg|jpeg|png|gif|webp|svg|ico)',
-  componentsSrc = afs.getComponents().map((comp) => ({ name: comp, path: path.join(afs.getComponentPath(comp), 'media', mediaGlobs) })),
-  layoutsSrc = afs.getLayouts().map((layout) => ({ name: layout, path: path.join(process.cwd(), 'layouts', layout, 'media', mediaGlobs) })),
-  styleguidesSrc = afs.getFolders(path.join(process.cwd(), 'styleguides')).map((styleguide) => ({ name: styleguide, path: path.join(process.cwd(), 'styleguides', styleguide, 'media', mediaGlobs) })),
-  sitesSrc = afs.getFolders(path.join(process.cwd(), 'sites')).map((site) => ({ name: site, path: path.join(process.cwd(), 'sites', site, 'media', mediaGlobs) }));
+  mediaGlobs = '*.+(jpg|jpeg|png|gif|webp|svg|ico)';
 
 /**
  * copy images and icons from components, layouts, styleguide, and sites folders
@@ -22,6 +18,11 @@ const _ = require('lodash'),
  * @return {Object} with build (Highland Stream) and watch (Chokidar instance)
  */
 function compile(options = {}) {
+  const componentsSrc = afs.getComponents().map((comp) => ({ name: comp, path: path.join(afs.getComponentPath(comp), 'media', mediaGlobs) })),
+    layoutsSrc = afs.getLayouts().map((layout) => ({ name: layout, path: path.join(process.cwd(), 'layouts', layout, 'media', mediaGlobs) })),
+    styleguidesSrc = afs.getFolders(path.join(process.cwd(), 'styleguides')).map((styleguide) => ({ name: styleguide, path: path.join(process.cwd(), 'styleguides', styleguide, 'media', mediaGlobs) })),
+    sitesSrc = afs.getFolders(path.join(process.cwd(), 'sites')).map((site) => ({ name: site, path: path.join(process.cwd(), 'sites', site, 'media', mediaGlobs) }));
+
   let watch = options.watch || false;
 
   gulp.task('media', () => {

--- a/lib/cmd/compile/templates.js
+++ b/lib/cmd/compile/templates.js
@@ -19,8 +19,6 @@ const _ = require('lodash'),
   destPath = path.join(process.cwd(), 'public', 'js'),
   templateGlob = 'template.+(hbs|handlebars)',
   variables = { minify: process.env.CLAYCLI_COMPILE_MINIFIED || process.env.CLAYCLI_COMPILE_MINIFIED_TEMPLATES },
-  componentPaths = afs.getComponents().map((name) => path.join(afs.getComponentPath(name), templateGlob)),
-  sourcePaths = componentPaths.concat([path.join(process.cwd(), 'layouts', '**', templateGlob)]),
   // bundles for group concatenating (when minifying)
   bundles = helpers.generateBundles('_templates', 'js'),
   getFileCtime = _.memoize((filepath) => fs.statSync(filepath).ctime);
@@ -177,6 +175,9 @@ function minifyTemplate(file, shouldMinify) {
  * @return {Object} with build (Highland Stream) and watch (Chokidar instance)
  */
 function compile(options = {}) {
+  const componentPaths = afs.getComponents().map((name) => path.join(afs.getComponentPath(name), templateGlob)),
+    sourcePaths = componentPaths.concat([path.join(process.cwd(), 'layouts', '**', templateGlob)]);
+
   let watch = options.watch || false,
     minify = options.minify || variables.minify || false;
 

--- a/package.json
+++ b/package.json
@@ -56,7 +56,7 @@
     "@babel/core": "^7.0.1",
     "@babel/preset-env": "^7.0.0",
     "@nymag/vueify": "^9.4.5",
-    "amphora-fs": "^1.0.1",
+    "amphora-fs": "^1.0.2",
     "autoprefixer": "^9.0.2",
     "babelify": "^10.0.0",
     "base-64": "^0.1.0",

